contracts/extensions/public-pools-strategy-manager.clar

```diff
@@ -2,9 +2,13 @@
 ;; SPDX-License-Identifier: BUSL-1.1
 
 (define-constant err-unauthorised (err u3000))
+(define-constant err-still-alive (err u3001))
+
+(define-constant dms-activation-period u12960) ;; ~90 days
```

Comment:
one cycle is already bad. So 2100 bitcoin blocks?

Reply:
Sure we can do that, but then a manager must be sure to send a `keep-alive` each cycle if there are no changes. The theory of the DMS was so that people can always get the underlying STX of their LiSTX back into the vault in case the managers go missing.

Reply:
Pool manager has to extend if nobody else does it for us